docs/adr/0003-allowing-implicit-object-type.md

```diff
@@ -0,0 +1,47 @@
+# Allowing implicit object type
+
+## Status
+
+Proposed.
```

Comment:
Should this change to accepted before we close the PR?

Reply:
Agreed